cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityOperatorTest.java

```diff
@@ -126,8 +126,17 @@ static Map<String, String> volumeMounts(List<VolumeMount> mounts) {
     private final EntityOperator entityOperator = EntityOperator.fromCrd(new Reconciliation("test", resource.getKind(), resource.getMetadata().getNamespace(), resource.getMetadata().getName()), resource, VERSIONS, SHARED_ENV_PROVIDER);
 
     @ParallelTest
-    public void testGenerateDeployment() {
+    public void testGenerateDeploymentWithBto() {
+        testGenerateDeployment(false);
+    }
 
+    @ParallelTest
+    public void testGenerateDeploymentWithUto() {
+        testGenerateDeployment(true);
+    }
+
+    private void testGenerateDeployment(boolean useUto) {
```

Comment:
not sure `useUto` is really understandable :-) I would use the more "verbose" version.